pageserver/src/tenant/storage_layer/image_layer.rs

```diff
@@ -879,12 +875,9 @@ impl ImageLayerWriterInner {
         // fsync the file
         file.sync_all().await?;
 
-        // FIXME: why not carry the virtualfile here, it supports renaming?
-        let layer = Layer::finish_creating(self.conf, timeline, desc, &self.path)?;
-
-        info!("created image layer {}", layer.local_path());
+        trace!("created image layer {}", self.path);
```

Comment:
I would prefer keeping this as a signal that `create_image_layers` compaction is running for a tenant when I look at the log, but I'd also be fine if this gets downgraded.

Reply:
I'm happy to keep it at INFO level if that's useful, but do you mind if it changes so that it prints the temporary path rather than the final filename? If we wanted to keep printing the final filename, that'd need a bit more refactoring.

Reply:
The final filename would be more useful, I can have another PR for it that prints the generated image layers at the end of compaction instead of here by the end of this week.

Reply:
Hmm, end of compaction sounds less useful as a progress indicator, right? A compaction might create lots of image layers.

Printing it in Layer::finish_creating would be one option

Reply:
I'll merge this as it is, changing the `info!` to `trace!`. @skyzh I'll leave it up to you to put it back or do something else in a follow-up PR if you want.